src/App.js

```diff
@@ -1,9 +1,23 @@
 import './Style.css';
+import { Routes, Route, BrowserRouter } from 'react-router-dom';
 import Calculator from './Components/Calculator';
+import Navbar from './Components/Nav';
+import HomePage from './Components/Home';
+import Quotes from './Components/Quotes';
```

Comment:
- [ ] I could not run your app properly due to this error in the image.

  ![image](https://user-images.githubusercontent.com/54848363/213338887-e61e3f86-49ff-47f4-9591-de0a3f00ae73.png)

  You need to be mindful of your file name and its casing. You have `components` with small letter, but you are trying to import using capitalized name `Components`. Kindly fix this.

  ```
  import Calculator from './components/Calculator';
  import Navbar from './components/Nav';
  import HomePage from './components/Home';
  import Quotes from './components/Quotes';
  ```

Reply:
working on that sir